### PR TITLE
Update organizations.yml

### PR DESCRIPTION
**Pull request by jkent2910**
Add Stitch fix and Bigger Pockets!

### CODE REVIEW

File: data/seeds/organizations.yml [hunk 1]
```diff
@@ -519,3 +519,14 @@
...
+  type: business
+  url: https://www.stitchfix.com 
+  twitter_handle: stitchfix
+  locations:
+  - address: 1 Montgomery Tower, San Francisco, CA 94104, USA
+- name: Bigger Pockets
+  type: business
+  url: https://www.biggerpockets.com 
+  twitter_handle: biggerpockets
+  - address: 3344 Walnut St, Denver, CO 80205, USA
```
Comment (lewispb) [inline data/seeds/organizations.yml:533]:
```suggestion
  locations:
    - address: 3344 Walnut St, Denver, CO 80205, USA
```

File: data/seeds/organizations.yml [hunk 2]
```diff
@@ -519,3 +519,15 @@
...
+  url: https://www.stitchfix.com 
+  twitter_handle: stitchfix
+  locations:
+  - address: 1 Montgomery Tower, San Francisco, CA 94104, USA
+- name: Bigger Pockets
+  type: business
+  url: https://www.biggerpockets.com 
+  twitter_handle: biggerpockets
+  locations:
+    - address: 3344 Walnut St, Denver, CO 80205, USA
```
Comment (lewispb) [inline data/seeds/organizations.yml]:
```suggestion
  - address: 3344 Walnut St, Denver, CO 80205, USA
```

Comment (jkent2910) [reply data/seeds/organizations.yml]: 🤦‍♀

**Comment by lewispb** [comment]
Awesome thanks @jkent2910 🤠

**Review by lewispb — APPROVED** [review]

File: data/seeds/organizations.yml (hunk 2 above)
Comment (lewispb) [reply data/seeds/organizations.yml]: it was my fault! made a dodgy suggestion 🤦‍♂

**Pull request merged**